src/components/VueBootstrapTypeaheadListItem.vue

```diff
@@ -28,19 +29,13 @@ export default {
     }
   },
 
-  data() {
-    return {
-      active: false
-    }
-  },
-
   computed: {
     textClasses() {
-      let classes = ''
-      classes += this.active ? 'active' : ''
-      classes += this.backgroundVariant ? ` bg-${this.backgroundVariant}` : ''
-      classes += this.textVariant ? ` text-${this.textVariant}` : ''
-      return `vbst-item list-group-item list-group-item-action ${classes}`
+      const classes = ['vbst-item', 'list-group-item', 'list-group-item-action']
```

Comment:
I noticed that `classes` is a const here, but it's being appended to (below)

Reply:
Yep, in es2015 the keyword `const` means that the variable won't be reassigned, not that the value is immutable. You can assign it to an Object (in this case an Array) and still set attributes later (or call methods that will set attributes).